code/modules/mob/living/carbon/human/life.dm

```diff
@@ -18,34 +18,33 @@
 #define THERMAL_PROTECTION_HAND_LEFT	0.025
 #define THERMAL_PROTECTION_HAND_RIGHT	0.025
 
-/mob/living/carbon/human/Life()
-	if (notransform)
+/mob/living/carbon/human/Life(delta_time = SSMOBS_DT, times_fired)
+	if(notransform)
```

Comment:
I don't really care myself, but for the future avoid changing these when you find them. It's purely a matter of personal taste and at a large enough scale it'll be you changing them one way, and someone else changing them back in 2 years. Better to just let it sit

Reply:
I mean, yes. But it's slightly compulsory at this point.

Reply:
Actually it's pure neutral, the guidelines straight up say not to touch it.